recipes/pastas/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pastas" %}
+{% set version = "0.9.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 374f63736150cf214da2a7024923d65cd24189aa67275196d62663336b44e1f8
+
+build:
```

Comment:
Can this be a `noarch:python` package? 
https://conda-forge.org/docs/meta.html#building-noarch-packages